pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -92,6 +92,15 @@ let
 
   browserPatches = [
     ./env_var_for_system_dir.patch
+    (fetchpatch {
+      url = "https://hg.mozilla.org/mozilla-central/raw-rev/f63ebd7e9e28";
+      sha256 = "0xmms4f3qxp6wfr76bqp8ymk14ksf86mmiwm2ka540zwqzhrc24b";
+    })
+    (fetchpatch {
+      url = "https://hg.mozilla.org/mozilla-central/raw-rev/4f2e84dc490d";
+      sha256 = "18ba9riwplchii60664j2shydb19s2gyd3jldr1225bqj3wky521";
+    })
+    ./fix-build-with-rust-1.33.patch
```

Comment:
Is there some source we could fetch the patch from or did you write it?

Reply:
I needed to adapt this patch, because the original did not apply.

Reply:
All fine with me. I'd just drop a line where it originally came from in case we have to follow up on that in the future.